PactNet/Mocks/MockHttpService/MockProviderService.cs

```diff
@@ -40,9 +41,13 @@ public class MockProviderService : IMockProviderService
             _httpMethodMapper = httpMethodMapper;
         }
 
-        public MockProviderService(int port, bool enableSsl, string providerName, PactConfig config)
+        public MockProviderService(int port, bool enableSsl, bool rewriteLocalhost, string providerName, PactConfig config)

```

Comment:
Can we please rename rewriteLocalhost to bindOnAllAdapters and arrange as the last param?